src/stcal/saturation/saturation.py

```diff
@@ -85,6 +84,20 @@ def flag_saturated_pixels(
             plane = data[ints, group, :, :]
 
             if read_pattern is not None:
```

Comment:
I don't see anywhere in the subsequent calls to `flag_saturated_pixels` that `read_pattern` is provided as an argument, so isn't it always going to be `None`?

Also, we already use the "readout pattern" nomenclature to refer to the *name* of a readout pattern, such as RAPID, DEEP8, IRS2, etc. Naming this variable "read_pattern" seems confusing, since as far as I can tell it's a list of readout times (which is totally different than what we normally think of when seeing "read pattern").

Reply:
Re the naming scheme, this is the name for this quantity in Roman, where of course there's another name for the Webb readout pattern.  Unfortunate collision.

Reply:
Gotcha. Since it's existing code, we on the JWST side can live with that. ;-)